### PR TITLE
collector intro page added

### PR DESCRIPTION
**Pull request by iusiri**

### CODE REVIEW

File: docs/pages/collectorintro.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+![Collector using SG Diagram](../assets/CollectorSGDiagram.jpeg)
+
+One Collector will run multiple Sample Generators at once - collecting and pushing Samples for each.
+
+
+Each Sample Generator provides a path to your target data source, a set of subjects and aspects that you want to send samples for and other contextual information used by the transform code to generate samples. 
+
+![Collector using SG Diagram](../assets/SGDiagram.jpeg)
+
+As is illustrated, the transform-code leverages user-provided value to context variables. Most users will never interact with the transform-code, instead, they will select the code template that works for them (more on this later!) and provide the context variables that are required.
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: no hyphen between transform and code

File: docs/pages/collectorintro.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+Each Sample Generator provides a path to your target data source, a set of subjects and aspects that you want to send samples for and other contextual information used by the transform code to generate samples. 
+
+![Collector using SG Diagram](../assets/SGDiagram.jpeg)
+
+As is illustrated, the transform-code leverages user-provided value to context variables. Most users will never interact with the transform-code, instead, they will select the code template that works for them (more on this later!) and provide the context variables that are required. 
+
+
+Getting onto Refocus has never been easier! Write a Sample Generator submit it to a Refocus instance and start visualizing!
```
Comment (iamigo) [inline docs/pages/collectorintro.md]: `Getting onto Refocus has never been easier!` --> `Getting started with Refocus has never been easier!`

`Write a Sample Generator submit it to a Refocus instance` --> `Write a Sample Generator, submit it to a Refocus instance`

**Comment by iamigo** [comment]
@iusiri

**Comment by iamigo** [comment]
@iusiri if you're still collecting feedback on the quip doc then please close this PR and open a new one when you have the updated content ready for review here.

File: docs/pages/collectorintro.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+#So, what is a Collector 
+
+A Collector is an out-of-the-box application (running outside of Refocus) which collects data from different data sources and sends Samples to Refocus. It does this by leveraging a user-provided config file called a “Sample Generator”.
+Most users will never have to maintain a collector (unless you choose to deploy your own), they'll write Sample Generators and send them to a shared Collector. 
+
+
+
+#Tell me more about Sample Generators
+
+To use a simple analogy, Collectors are akin to the “human body”, they do the manual heavy-lifting, but every body (and indeed everybody!) needs a brain in order to tell it what to do - the Sample Generator is that “brain”.
```
[the remark this answers is not in the document]
Comment (iusiri) [reply docs/pages/collectorintro.md]: No big difference here. Will leave as is.

File: docs/pages/collectorintro.md (hunk 1 above)
Comment (iusiri) [reply docs/pages/collectorintro.md]: done.

File: docs/pages/collectorintro.md (hunk 2 above)
Comment (iusiri) [reply docs/pages/collectorintro.md]: updated.

File: docs/pages/collectorintro.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+#Introduction
+
+Refocus makes it easy for users to understand the real-time status of their systems. To date, achieving this has required users to develop API integrations in order to push Samples into Refocus. This process had a few limitations: it required developers to write api integrations, and, at times, meant teams had to operate an extra 'always-on' service. 
+
+Collectors are designed to tackle these issues and make it easier than ever to leverage all the capabilities of Refocus!
+
+
+#So, what is a Collector 
+
+A Collector is an out-of-the-box application (running outside of Refocus) which collects data from different data sources and sends Samples to Refocus. It does this by leveraging a user-provided config file called a “Sample Generator”.
```
[the remark this answers is not in the document]
Comment (iusiri) [reply docs/pages/collectorintro.md]: done

File: docs/pages/collectorintro.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+
+#Introduction
+
+Refocus makes it easy for users to understand the real-time status of their systems. To date, achieving this has required users to develop API integrations in order to push Samples into Refocus. This process had a few limitations: it required developers to write api integrations, and, at times, meant teams had to operate an extra 'always-on' service. 
+
+Collectors are designed to tackle these issues and make it easier than ever to leverage all the capabilities of Refocus!
+
+
+#So, what is a Collector
```
[the remark this answers is not in the document]
Comment (iusiri) [reply docs/pages/collectorintro.md]: I don't think the headers should have ending punctuation. Also not sure about capitalizing the "w" for "what"

File: docs/pages/collectorintro.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+Refocus makes it easy for users to understand the real-time status of their systems. To date, achieving this has required users to develop API integrations in order to push Samples into Refocus. This process had a few limitations: it required developers to write api integrations, and, at times, meant teams had to operate an extra 'always-on' service. 
+
+Collectors are designed to tackle these issues and make it easier than ever to leverage all the capabilities of Refocus!
+
+
+#So, what is a Collector 
+
+A Collector is an out-of-the-box application (running outside of Refocus) which collects data from different data sources and sends Samples to Refocus. It does this by leveraging a user-provided config file called a “Sample Generator”.
+Most users will never have to maintain a collector (unless you choose to deploy your own), they'll write Sample Generators and send them to a shared Collector.
```
[the remark this answers is not in the document]
Comment (iusiri) [reply docs/pages/collectorintro.md]: I'm not sure what you mean here.

File: docs/pages/collectorintro.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (iusiri) [reply docs/pages/collectorintro.md]: Done.

**Comment by iusiri** [comment]
New commit. Please review.

**Comment by iamigo** [comment]
Is this ready for review now? If so, please reopen.

**Pull request closed without merging**